doc/languages.rst

```diff
@@ -549,6 +549,10 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       Check syntax and lint with `ESLint <http://eslint.org/>`_.
 
+      .. note::
+
+         This syntax checker requires the presence of a configuration file.
```

Comment:
I think "Flycheck does not use this syntax checker if eslint cannot find a valid configuration file." is more precise, isn't it?